src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -5234,6 +5234,26 @@ private static (ulong val, bool ovf) ShiftOvf(ulong value, int shift)
 
         public static float MinNumberPairwise(float[] op1, float[] op2, int i) => Pairwise(MinNumber, op1, op2, i);
 
+        public static float[] MultiplyAddRotateComplex(float[] op1, float[] op2, float[] op3, byte imm)
+        {
+            for (int i = 0; i < op1.Length; i += 2)
+            {
+                (float ans1, float ans2) = imm switch
+                {
+                    0 => (FusedMultiplyAdd(op1[i], op2[i], op3[i]), FusedMultiplyAdd(op1[i + 1], op2[i], op3[i + 1])),
```

Comment:
I tried this, and I'm getting test failures due to slight imprecision in the arithmetic operations. For example, I'm getting results like this:
```
Helpers result: (0.18111514, 0.7438505, 0.7731012, 0.6613971)
API result: (0.18111514, 0.7438505, 0.7731013, 0.66139716)
```

The docs for `Math.FusedMultiplyAdd` have the following remark:
> This computes (x * y) as if to infinite precision, adds z to that result as if to infinite precision, and finally rounds to the nearest representable value.
> This differs from the non-fused sequence which would compute (x * y) as if to infinite precision, round the result to the nearest representable value, add z to the rounded result as if to infinite precision, and finally round to the nearest representable value.

So I think it might be simpler to stick to using `FusedMultiplyAdd` in the helpers.